circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -322,7 +322,11 @@ def find_cuts(
             gate_ids.append(action[1][0])
         else:
             # We only support three types of optimizer cuts
```

Comment:
Did you mean three types of _wire_ cuts? I would personally be inclined to either remove this comment or change it to something like ''We only support three actions in the context of wire cutting.''